lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1280,60 +1282,6 @@ static char getTypeSuffix(Type *T) {
   return Suffix;
 }
 
-// ToDo: Handle unsigned integer return type. May need spec change.
```

Comment:
This `TODO` was lost, I suggest that we preserve it if it is still actual

Reply:
Ok, fair enough.